src/components/modules/saver.ts

```diff
@@ -104,15 +104,14 @@ export default class Saver extends Module {
         return;
       }
 
-      const output: any = {
+      const output = {
         type: tool,
         data,
+        ...!_.isEmpty(tunes) && {
+          tunes,
+        },
```

Comment:
`Conditional spread` helps the type definition!

https://medium.com/@slamflipstrom/conditional-object-properties-using-spread-in-javascript-714e0a12f496